backend/src/main/kotlin/blue/mild/covid/vaxx/dao/model/PatientDataCorrectnessConfirmation.kt

```diff
@@ -0,0 +1,28 @@
+package blue.mild.covid.vaxx.dao.model
+
+object PatientDataCorrectnessConfirmation : ManagedTable("patient_data_correctness") {
```

Comment:
`correctness nebo spis `corrections` ? 🤔

Reply:
no to je otazka - jedna se o potvrzeni toho, ze data jsou spravne a ze je zkontroloval ten a ten, to bych si myslel ze je `correctness`? mozna ne, jistej si uplne nejsem